client/js/widgets/widget.js

```diff
@@ -15,7 +15,8 @@ const readOnlyProperties = new Set([
   '_centerAbsoluteX',
   '_centerAbsoluteY',
   '_localOriginAbsoluteX',
-  '_localOriginAbsoluteY'
+  '_localOriginAbsoluteY',
+  '_totals' // Property of scoreboards
```

Comment:
Is this needed here? Feels weird to have it in here. It also would affect `_totals` of non-scoreboard widgets, right?

Reply:
It does feel weird, I agree. This is the first widget-specific readonly property in VTT. But without putting it here, `SET` will go right ahead and try to (and succeed) set the `_totals` property. `get` will work fine without it, since it would then just call `super`. If you have a better suggestion, I'm happy to use it.

Reply:
Ah, `set` is the reason. I'm not sure if this is better or overriding `set` in `ScoreBoard` and ignoring calls for `_totals` (basically duplicated code).

A cleaner solution would probably be to call a `getTypeSpecificReadonlyProperties` function from `widget.js` which is defined in `scoreboard.js`.

Reply:
Much better idea. Done.